### DISCUSSION TITLE
About the Assistans

### DISCUSSION THREAD

**Discussion by MayDay-wpf** [betalgo/openai#514] [General]
I hope to join the Assistans☺️

**Comment by kayhantolga** [comment]
Thank you. Currently, there are two open pull requests. I will be reviewing the code and merging one of them. If you want, you're welcome to join the code review and leave a comment if you notice anything.

**Reply by Nicelittlewood** [reply]
@kayhantolga, recently OpenAi adding Streaming support to Assistant' API. 
Assistant is still in Beta, but with this addition i think there's nothing more they will add before exit the BETA. 
It could be the true big addition for the library.

**Comment by kayhantolga** [comment]
https://github.com/betalgo/openai/pull/535 Here is a PR if anyone wants to do a code review.
